change/@ni-nimble-angular-b3775054-4fc0-4ca5-abdf-18a89277520c.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Reminder to update the Angular version mentioned in the `nimble-angular` README.